### PR TITLE
Update example to be unambiguous

### PR DESCRIPTION
**Pull request by AnilSeervi**
- #1890 didn't actually fix the issue originally created at mdn/content#7975
- I've changed the example to match the suggestion of the OP

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
I wasn't awake when reviewing the other PR.

File: live-examples/js-examples/number/number-min-value.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-function multiply(x, y) {
-  if (x * y < Number.MIN_VALUE) {
-    return 'Process as 0';
+function divide(x, y) {
+  if (x / y < Number.MIN_VALUE) {
+    return `Process as 0`;
```
Comment (teoli2003) [inline live-examples/js-examples/number/number-min-value.js:3]: Don't use backticks here, but single quotes.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by AnilSeervi** [comment]
> I wasn't awake when reviewing the other PR.

NP, I was waiting that the OP would do the changes.

File: live-examples/js-examples/number/number-min-value.js (hunk 1 above)
Comment (AnilSeervi) [reply live-examples/js-examples/number/number-min-value.js:3]: I was testing out another approach, Sry I left it that way.
I'll make the changes and ping you.

Comment (AnilSeervi) [reply live-examples/js-examples/number/number-min-value.js]: DONE!

**Comment by MadaraUchiha** [comment]
Apologies <3 I meant to do this and the responses to my original issue were lost in the many emails from GitHub I get. Thanks for picking up my slack <3

**Comment by AnilSeervi** [comment]
> Apologies <3 I meant to do this and the responses to my original issue were lost in the many emails from GitHub I get. Thanks for picking up my slack <3

No Problem.

**Comment by himanshugarg** [comment]
This was merged 3 weeks ago but doesn't seem to be live yet. See also [ mdn issue 2372 ](https://github.com/mdn/content/issues/2372)

**Pull request merged**